lib/reversetunnel/srv.go

```diff
@@ -629,7 +630,7 @@ func (s *server) handleTransport(sconn *ssh.ServerConn, nch ssh.NewChannel) {
 // TODO(awly): unit test this
```

Comment:
Is it still worth it to keep this TODO around?

Reply:
This PR doesn't change its relevance.  Leaving it in for now.